documentation/business-analyst-activities.md

```diff
@@ -1 +1,19 @@
 # Business Analyst Activities
+
+You must have a GitHub account and access to the Teaching Vacancies repository to be able to make any changes to the code base, or to edit/create files (e.g, .yml or .md text files)
+<Insert link here to new page> This file explains the process by which new team members can get the access they require on GitHub.
+
+## Updating Polygon Mappings in the mapped_locations.yml file
+This section details the process for updating the search mappings so that a polygon search is used in place of a point search.
+
+1. Navigate to the "Code" tab within github and press "go to file"
+2. Search for [mapped_locations.yml](https://github.com/DFE-Digital/teaching-vacancies/blob/master/lib/tasks/data/mapped_locations.yml) and click on the file.
+  * This file contains a list of all of the mappings that have been applied, in alphabetical order.
+3. Navigate to the desired row, and input your new mapping in the following format:
+  * -['location that is searched by user', 'name of polygon we want to return']
+  * The automated tests are useful for confirming that all rows have been entered correctly. If one of these tests fails check that all "'" and "," have been entered correctly. On the PR, in the merging section at the bottom of the page there is a section dedicated to automated checks. This will show any checks that may have failed as part of the changes. ![Automated tests screenshot](https://user-images.githubusercontent.com/72141/103927287-0d570500-5112-11eb-902d-5d36a1c7e10a.png)
+  * The polygon names can be found (here)[https://github.com/DFE-Digital/teaching-vacancies/tree/master/lib/tasks/data]
+4. Once all desired mappings have been added, scroll to the bottom of the page and update the fields in the "Commit changes" box. This should include a concise title for the change, and further details of what you have done in the extended desription field.
+  * Once you have added this information ensure that the "Create a new branch for this commit and start a pull request." radio button is selected and press "Commit changes".
+5. Before the change can commited to >master. it must be approved by another member of the team and pass a set of automated tests. Once these steps have been completed press "Merge pull request" and your change will be committed to the production environment.
```

Comment:
`>master.` this punctuation is confusing!

Reply:
Not sure what formatting I was going for there, have changed it to master just to make it more clear.